### DISCUSSION TITLE
Bool type conversion error in encode method

### DISCUSSION THREAD

**Discussion by JackTong0312** [stephenh/ts-proto#641] [Q&A]
required bool dynamic_screen_size = 5 [default = true];

 encode

  if (message.dynamicScreenSize === true) {
      writer.uint32(40).bool(message.dynamicScreenSize);
    }

**Comment by stephenh** [comment]
Hi @tongxuerui , sorry, but I don't know what the error is just from the snippet you included.

It looks like we were missing a `bool` example in our main "simple" integration test, but I've added it here and it seems to be working fine:

https://github.com/stephenh/ts-proto/pull/646
